spec/project_spec.rb

```diff
@@ -109,11 +109,12 @@ class InvalidErrorHandler; end
     it 'should log an error when provided an invalid JSON datafile and skip_json_validation is true' do
       expect_any_instance_of(Optimizely::SimpleLogger).to receive(:log).once.with(Logger::ERROR, 'Provided datafile is in an invalid format.')
 
-      Optimizely::Project.new('{"foo": "bar"}', nil, nil, nil, true)
+      Optimizely::Project.new('{"foo": "bar"}', nil, nil, nil, false)
```

Comment:
you don't need to make it true. Leave existing test case as it is.

Reply:
Its invalid json, to raise InvalidInputError exception we need to set it false.
Otherwise it will get InvalidDatafileVersionError.
.
' expected: (3, "Provided datafile is in an invalid format.")
 got: (3, "This version of the Ruby SDK does not support the given datafile version: .")'